city_scrapers/spiders/chi_ssa_16.py

```diff
@@ -0,0 +1,105 @@
+import datetime
+import re
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa16Spider(CityScrapersSpider):
+    name = "chi_ssa_16"
+    agency = "Chicago Special Service Area #16"
+    timezone = "America/Chicago"
+    start_urls = ["https://greektownchicago.org/about/ssa-16/"]
+
+    def parse(self, response):
+        # Meeting entries are contained in a list item without any specific class or id designation.
+        # Because of this, I filtered the present list items on the page with the ", 20" string
+        # (every meeting li has this in the present text from the year listed),
+        # which indicates the meeting occurred or is scheduled to occur
+        # and therefore contains relevant information to document.
+
+        for item in response.xpath('//li[contains(text(), ", 20")]'):
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self._parse_location(item),
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        # There was no variation in types of meetings, so this was applicable for all.
+        return "Greektown Special Service Area Tax Commission #16 Public Meeting"
+
+    def _parse_description(self, item):
+        # This extracts the useful text provide to describe the services provided by the SSA.
```

Comment:
The `description` field is used for unique information about individual meetings since it's easier to store information about agencies separately so this can be left blank